Source/Core/Core/PowerPC/JitArm64/Jit.cpp

```diff
@@ -363,18 +363,35 @@ void JitArm64::WriteExit(u32 destination, bool LK, u32 exit_address_after_return
 
   if (LK)
   {
-    // Push {ARM_PC+20; PPC_PC} on the stack
+    // Push {ARM_PC; PPC_PC} on the stack
     MOVI2R(ARM64Reg::X1, exit_address_after_return);
-    ADR(ARM64Reg::X0, 20);
+    ADR(ARM64Reg::X0, JitArm64BlockCache::BLOCK_LINK_FAST_BL_OFFSET + 12);
```

Comment:
The 12 is a bit of a magic value, it's used in a few places, does it warrant a macro/constexpr or some ASSERT that the distance to after the BL is still 12, an it could maybe also be put in terms of 3 * sizeof-arm64-instruction?

Reply:
I've rewritten this one as `JitArm64BlockCache::BLOCK_LINK_SIZE + sizeof(u32) * 2`, which should hopefully make it clearer. The 2 is for the 2 instructions here before the block link: The ADR and the STP. I've added asserts too.